src/components/VenueSearchBar/VenueSearchBar.js

```diff
@@ -1,19 +1,29 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
 import * as actions from '../../actions';
 import requests from '../../utils/requests/requests';
 
 export function VenueSearchBar(props) {
-  let cityInput;
+  const { location } = props;
+  const [city, handleChange] = useState('');
+  const [redirect, setRedirect] = useState(false);
```

Comment:
nice hooks !

Reply:
Yup! I think hooks are alot easier to read and work with plus less code!